src/content/docs/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-ui.mdx

```diff
@@ -157,3 +157,10 @@ Various pages include a metrics or spans toggle. This allows you to choose wheth
 Metrics are not subject to sampling, and are thus more accurate, especially when computing rates like throughput. However, metrics are subject to cardinality constraints, and may lack certain attributes important for analysis. In contrast, spans are sampled and thus subject to accuracy issues, but have richer attributes since they are not subject to cardinality constraints.
 
 Historically, OpenTelemetry language APIs and SDKs and instrumentation prioritized trace instrumentation. However, the project has come a long way and metrics are available in almost all languages. Check the [documentation](https://opentelemetry.io/docs/languages/) of the relevant language and instrumentation for more details.
+
+## Golden metrics
+Golden metrics are a set of low-cardinality copies of golden signals data (e.g. HTTP/RPC metrics) that are used to populate many experiences across the platform including the Entity Explorer, Workloads Activity page, and Change Tracking Details page, among others. These metrics all use names like “newrelic.goldenmetrics.ext.service.*”
```

Comment:
Golden metrics are low-cardinality versions of golden signals data, such as HTTP/RPC metrics. They populate various platform experiences, including the Entity Explorer, Workloads Activity page, and Change Tracking Details page. These metrics use names like newrelic.goldenmetrics.ext.service.*.